metadata.yaml

```diff
@@ -0,0 +1,14 @@
+flavours:
```

Comment:
**praise** I really like the approach and the idea to use a `metadata.yaml` file to provide the details about the different images. I would possibly also add a key `build_file` where we provide the name of the dockerfile to be used. If needed, we could also use this file to custom the build of OCI images (e.g. adding tags to the image), but not needed right now. 

However in general, maybe we could make the build a bit more parametric if possible with respect to the flavour, such that if we need to add a new image, this only means:

1. Adding a Dockerbuild or rockcraft.yaml
2. Adding the YAML 

Hopefully without touching the makefile